components/profile/notification-form/view.tsx

```diff
@@ -127,6 +133,16 @@ export default function NotificationFormView({
             </div>
           </div>
         </If>
+
+        <If condition={isEmailConfirmed && !emailVerificationError}>
+          <div className="row align-items-center mt-2">
+            <div className="col">
+                <span className="text-info xs-medium font-weight-normal">
+                  {t("profile:notifications-form.email-confirmed")}
+                </span>
+            </div>
```

Comment:
I think we can improve this to aline with the UI guidelines, don't forget to add the import of `DoneIcon` on top with `import DoneIcon from "assets/icons/done-icon";`
```suggestion
            <div className="col">
              <span className="text-green-500 xs-medium">
                <DoneIcon className="mr-1" />
              </span>
              <span className="ml-5 text-green-500 xs-medium font-weight-normal">
                {t("profile:notifications-form.email-confirmed")}
              </span>
            </div>
```

Preview

![CleanShot 2024-09-06 at 18 40 33@2x](https://github.com/user-attachments/assets/55e42720-1154-4f21-a51f-f501b194a26c)